Mainnet/DividendToken/DividendToken/DividendToken.cs

```diff
@@ -0,0 +1,288 @@
+﻿using Stratis.SmartContracts;
+using Stratis.SmartContracts.Standards;
+
+[Deploy]
+public class DividendToken : SmartContract, IStandardToken
+{
+
+    public ulong Dividends
+    {
+        get => PersistentState.GetUInt64(nameof(this.Dividends));
+        private set => PersistentState.SetUInt64(nameof(this.Dividends), value);
+    }
+
+    private Account GetAccount(Address address) => PersistentState.GetStruct<Account>($"Account:{address}");
+
+    private void SetAccount(Address address, Account account) => PersistentState.SetStruct($"Account:{address}", account);
+
+
+    public DividendToken(ISmartContractState state, ulong totalSupply, string name, string symbol)
```

Comment:
Per Issue https://github.com/stratisproject/CirrusSmartContracts/issues/29, I believe `totalSupply` and other token value related fields should be updated with this issue proposal to match ETH's solidity `UInt256` token based values for standardization amongst token types and wrapped Eth tokens.